### ISSUE TITLE
example in get_site_MODIS function

### ISSUE THREAD

**Issue by oggioniale** [oggioniale/ReLTER#41]
@micha-silver 
I don't understand the example currently in the get_site_MODIS function
```
#'  \dontrun{
#'  deimsid = "https://deims.org/86e3a1ca-b2ba-4b06-b096-71447df52841"
#'  product_bands <- MODIStsp_get_prodlayers("M*D13Q1")
#'  print(product_bands[,6:7])
#' }
```

Please can you specify better the example?

**Comment by micha-silver** [comment]
Yes, I'll get to that over the weekend.

**Comment by micha-silver** [comment]
I've pushed a better example.

**Comment by micha-silver** [comment]
Yes, why not?
After tomorrow ;-)

**Comment by oggioniale** [comment]
@micha-silver
do you think of an internal function? 
This could be a good solution also because some functions, currently, don't have the show_map parameter.
My doubts is about to the abstraction of the values contained in the `relter_display_map()` function and how to activate this in other functions.

**Comment by oggioniale** [comment]
@micha-silver if you have more clear that me how to build this please. But take care that a strong modification to the functions must take into account the test, and probably, any modifications must consider the test files.
I suggest to propose modification in a new branch that cover specific issue.

**Comment by micha-silver** [comment]
Good idea, I'll start a new branch `dev-display-maps` for this and begin to make changes. Then we'll see how it goes...
